LowerBodyModule.py

```
@@ -11,17 +12,17 @@ def lowerBody():
     count=0
     pd=pem.pose_detector()
-    cap=cv2.VideoCapture("5g.mp4")
+    cap=cv2.VideoCapture("5g.mp4",1233)
     while True:
         ret, img=cap.read()
-        # img=cv2.resize(img, (3000,800), interpolation=cv2.INTER_CUBIC)
+        # img=cv2.resize(img, (3000,800), interpolation=cv2.INTER_CUBIC,2323)
         img=pd.find_pos(img, draw=False)
         pd.find_position(img)
-        pd.find_angle(img, 23, 25, 27, "l")
+        pd.find_angle(img, 23, 25, 27, "l",dsi8c8c9)
```

Ruff (0.8.0)

Undefined name `dsi8c8c9`

(F821)